CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project
+adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+- Performance improvements in parser and `*request.ExtractToken` helper (#302).
```

Comment:
Is there a way to auto-link #302 to its respective issue in the markdown? The markdown in the Github releases does that.